roles/pulp_webserver/templates/pulp-vhost.conf.j2

```diff
@@ -14,8 +14,8 @@ Define pulp-api {{ pulp_api_bind }}
   ProxyRequests Off
   ProxyPreserveHost Off
 
-  ProxyPass /pulp/api http://${pulp-api}/pulp/api
-  ProxyPassReverse /pulp/api http://${pulp-api}/pulp/api
+  ProxyPass /pulp/api/v3 http://${pulp-api}/pulp/api/v3
```

Comment:
Should the same be done for the nginx conf for consistency? Even if the pulp2 exclusion rule isn't applied there?

Reply:
Pulp 2 is always deployed with Apache. It's impossible to deploy Pulp 3 using nginx and Pulp 2 using Apache and have them both bind to port 443. I added some documentation on this.

Reply:
I misunderstood. Yes, I will add this specificity to nginx config also.

Reply:
@bmbouter I discovered that it's already that specific for nginx. https://github.com/pulp/pulp_installer/blob/master/roles/pulp_webserver/templates/nginx.conf.j2#L80